docs/en/enterprise-edition/policy-reference/azure-policies/azure-secrets-policies/bc-azure-239.adoc

```diff
@@ -0,0 +1,68 @@
+
+== Azure Synapse Workspace administrator login password is exposed
```

Comment:
Options

-  Azure Synapse workspace administrator login password exposed
- Exposed administrator login password in Azure Synapse workspace